docs/UserGuide.md

```diff
@@ -137,11 +137,14 @@ _Help window displayed after running the `help` command_
 
 Adds a patient to the patient list.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [d/TASK_DESCRIPTION]… [t/TAG]…`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [d/TASK_DESCRIPTION | DATE TIME]… [t/TAG]…`
+
+* Date and time should be in the format like 16-10-2022 10:30
```

Comment:
Just a suggestion on time format: perhaps we could use 24h time without the colon? e.g. 1030 instead of 10:30 because it's an extra shift key to press.

Reply:
sure will change it